src/V3Tristate.cpp

```diff
@@ -1412,11 +1412,6 @@ class TristateVisitor final : public TristateBaseVisitor {
         }
     }
     void visitEqNeqWild(AstNodeBiop* nodep) {
-        if (!VN_IS(nodep->rhsp(), Const)) {
-            nodep->v3warn(E_UNSUPPORTED,  // Says spac.
-                          "Unsupported: RHS of ==? or !=? must be constant to be synthesizable");
-            // rhs we want to keep X/Z intact, so otherwise ignore
-        }
```

Comment:
I don't know if it is possible to enter into the visit method of `==?` only in one of these phases. I thought that it would be enough to have this warning only in one phase.
I restored this warning in V3Tristate, but added the same condition as in V3Unknown.